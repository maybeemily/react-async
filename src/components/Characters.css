.Characters {
  display: flex;
  flex-wrap: wrap;
  justify-content: space-around;
  align-items: center;

  ul {
    margin-bottom: 4vw;
  }

  li {
    list-style-type: none;
    text-align: center;
    font-family: sans-serif;
  }
  
  img {
    width: 20vw;
  }
}
