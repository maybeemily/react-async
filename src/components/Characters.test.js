import React from 'react';
import { shallow } from 'enzyme';
import Characters from './Characters';

describe('Characters component test', () => {
  it('renders characters', () => {
    const charactersArray = [
      {
        image: '/path',
        name: 'Morty',
        status: 'Alive',
        species: 'Human'
      },
      {
        image: '/path',
        name: 'Rick',
        status: 'Alive',
        species: 'Human'
      }
    ];
    const wrapper = shallow(<Characters characters = {charactersArray}/>);
    expect(wrapper).toMatchSnapshot();
  });
});
