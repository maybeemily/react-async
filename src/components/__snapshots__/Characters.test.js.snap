// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Characters component test renders characters 1`] = `
<ul>
  <li
    key="0"
  >
    <Character
      character={
        Object {
          "image": "/path",
          "name": "Morty",
          "species": "Human",
          "status": "Alive",
        }
      }
    />
  </li>
  <li
    key="1"
  >
    <Character
      character={
        Object {
          "image": "/path",
          "name": "Rick",
          "species": "Human",
          "status": "Alive",
        }
      }
    />
  </li>
</ul>
`;
